package.json

```diff
@@ -83,10 +83,12 @@
     "libphonenumber-js": "^1.10.60",
     "nunjucks": "^3.2.4",
     "openid-client": "^5.6.5",
-    "pino": "^8.20.0",
+    "otplib": "^12.0.1",
+    "pino": "^8.19.0",
     "pino-http": "^9.0.0",
     "prettier": "^3.2.5",
-    "uuid": "^9.0.1",
```

Comment:
Can't we use uuid 9 instead?

Reply:
yes, i'm not sure how it got downgraded.